test/integration/developer_portal/signup_test.rb

```diff
@@ -26,6 +26,48 @@ def test_create
     assert_response :redirect
   end
 
+  # A post request can't be sent directly to the controller bypassing the form
+  # when the spam protection is always enabled.
+  def test_create_spam_protected_always
+    @provider.settings.update(spam_protection_level: :captcha)
+    DeveloperPortal::SignupController.any_instance.stubs(:verify_captcha).returns(false)
+
+    post signup_path(account: {
+                       promo_code: '12345',
+                       org_name:   'alaska',
+                       user: {
+                         email:    'foo@example.edu',
```

Comment:
To reduce potential typos/false positives, remove hardcoded values by extracting the target string into a variable `test_email = 'foo@example.edu'`.

Reply:
Done